### ISSUE TITLE
[ActionSheetIOS] is undefined React 0.7

### ISSUE THREAD

**Issue by grabbou** [facebook/react-native#2110]
ActionSheetIOS is undefined when imported like below:
```
import React, {ActionSheetIOS, View} from 'react-native'
```
It's also undefined when accessed via `React` like so `React.ActionSheetIOS` while both React and View are OK. 

Writing `import ActionSheet from 'ActionSheetIOS'` works normally. Weird, given last issues and hints from @ide that all important packages were going to be moved into React namespace by default.

**Comment by ide** [comment]
Fixed in #https://github.com/facebook/react-native/commit/14fef6474df5437e3447eb135f587ea8b5795ec9. The plan is for 0.8.0 to go out later today.

**Comment by grabbou** [comment]
Sorry for that, hard to follow up :(
